labelCloud/control/controller.py

```diff
@@ -244,27 +247,26 @@ def key_press_event(self, a0: QtGui.QKeyEvent) -> None:
         """Triggers actions when the user presses a key."""
 
         # Reset position to intial value
-        if a0.key() == QtCore.Qt.Key_Control:
+        if a0.key() in [Keys.Key_Control]:
```

Comment:
Why did you make it a list?

Reply:
I used lists everywhere to easily add alternative keys on demand, but it is not huge deal, so I can delete it if you want.